### PR TITLE
Make users and funds templates translatable.

### PR DESCRIPTION
**Pull request by frjo**

### CODE REVIEW

File: hypha/apply/users/templates/users/activation/email.txt [hunk 1]
```diff
@@ -1,8 +1,10 @@
-{% load wagtailadmin_tags %}
+{% load i18n wagtailadmin_tags %}
 {% base_url_setting as base_url %}
-Dear {{ name|default:username }},
+{% name|default:username as user %}
```
Comment (frjo) [inline hypha/apply/users/templates/users/activation/email.txt]: @Parbhat This seems not to work, suggestions?

I want the translation string below to be the same as in other places to you only need to translate it once.

File: hypha/apply/users/templates/users/activation/email.txt [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 
-An account on the {{ ORG_LONG_NAME }} web site has been created. Activate your account by clicking this link or copying and pasting it to your browser:
+{% blocktrans %}Dear {{ user }},{% endblocktrans %}
```
Comment (frjo) [inline hypha/apply/users/templates/users/activation/email.txt:5]: Want it to look like this and not `{% blocktrans %}Dear {{ name|default:username }},{% endblocktrans %}`.

File: hypha/apply/users/templates/users/activation/email.txt (hunk 1 above)
Comment (frjo) [reply hypha/apply/users/templates/users/activation/email.txt]: @sks444 or @sandeepsajan0 Suggestion for my issue above?

Comment (sandeepsajan0) [reply hypha/apply/users/templates/users/activation/email.txt]: @frjo  I am checking , I think we need to use 'with' to make it available inside a block. Let me check more options.

Comment (sandeepsajan0) [reply hypha/apply/users/templates/users/activation/email.txt]: @frjo can you try it like this `{% firstof  name username as user %}`.  I think you wants to show `name` and if name is None then `username`, right?

Comment (frjo) [reply hypha/apply/users/templates/users/activation/email.txt]: Thank you @sandeepsajan0, implemented your fix now.

**Pull request merged**